app/src/main/kotlin/com/wire/android/ui/home/conversations/details/options/GroupConversationOptionsState.kt

```diff
@@ -14,17 +14,21 @@ data class GroupConversationOptionsState(
     val isServicesAllowed: Boolean = false,
     val isUpdatingAllowed: Boolean = false,
     val isUpdatingGuestAllowed: Boolean = false,
-    val isAbleToRemoveGroup: Boolean = true,
     val changeGuestOptionConfirmationRequired: Boolean = false,
     val changeServiceOptionConfirmationRequired: Boolean = false,
     val loadingGuestOption: Boolean = false,
     val loadingServicesOption: Boolean = false,
-    val isSelfUserMember: Boolean = true,
-    val error: Error = Error.None
+    val error: Error = Error.None,
 ) {
+
     sealed interface Error {
         object None : Error
         class UpdateGuestError(val cause: CoreFailure) : Error
         class UpdateServicesError(val cause: CoreFailure) : Error
     }
 }
+
+sealed class BottomSheetContent {
+    data class Conversation(val conversationData: ConversationSheetContent) : BottomSheetContent()
+    data class Mute(val conversationData: ConversationSheetContent) : BottomSheetContent()
```

Comment:
Since we have another content called `ChangeRole` what do you think of getting this uniformed, maybe `ChangeNotification` or `MuteConversation` or `Notification`, just an idea for this Mute option to be more self-explanatory :D

Reply:
actually all this `BottomSheetContent ` is not used, removed it